src/patternfly/components/Select/select.scss

```diff
@@ -257,7 +273,10 @@
 
 .pf-c-select__toggle-clear + .pf-c-select__toggle-arrow {
```

Comment:
Does this selector apply anymore? I don't see it in the DOM.

Reply:
Ah I forgot to remove these, thanks!